validator/accounts/accounts_helper.go

```diff
@@ -42,10 +42,14 @@ func selectAccounts(selectionPrompt string, pubKeys [][fieldparams.BLSPubkeyLeng
 	exit := "Done selecting"
 	results := make([]int, 0)
 	au := aurora.NewAurora(true)
+	if len(pubKeyStrings) > 5 {
+		log.Warnf("there are more than %d potential public keys to exit, please consider using the --%s or --%s flags", 5, flags.VoluntaryExitPublicKeysFlag.Name, flags.ExitAllFlag.Name)
```

Comment:
5 feels like a bit of a magic number, is there a constant we can use for this?

Reply:
It's a magic number in that library we are using if you look in that promptui not sure if it changes on updates